code/go/0chain.net/blobbercore/handler/handler.go

```diff
@@ -125,6 +125,18 @@ func setupHandlerContext(ctx context.Context, r *http.Request) context.Context {
 	return ctx
 }
 
+func setupGRPCMuxHandlerContext(ctx context.Context, r *http.Request, pathParams map[string]string) context.Context {
```

Comment:
Why? We already have a similar function, look at the functions at `grpc_handler.go`

Reply:
Yes, this to get path params or we need to change that function to get map[string]string

Reply:
We already have function to do this see this line in the `grpc_handlers.go` `setupGrpcHandlerContext(ctx, getGRPCMetaDataFromCtx(ctx))`